### PR TITLE
Change config file

### PR DESCRIPTION
**Pull request by qgan7125**

### CODE REVIEW

File: isb/Dockerfile [hunk 1]
```diff
@@ -15,7 +15,8 @@ COPY ./isamples_webui/ ./
 ARG ISB_SITEMAP_PREFIX=https://mars.cyverse.org
 ARG ANALYTICS_DOMAIN=isamples.org
 # write out the JSON config for the specific solr URL to hit in the webUI
-RUN echo "{\n    \"solr_url\": \"$ISB_SITEMAP_PREFIX/thing/select\",    \"analytics_src\": \"https://metrics.isample.xyz/js/plausible.js\",\n    \"analytics_domain\": \"$ANALYTICS_DOMAIN\"\n}" | tee ./src/config.json
+RUN echo "const _solr_base = \"https://hyde.cyverse.org/isamples_central/thing/\";\nexport default {\n  \"solr_url\": _solr_base + \"select\",\n  \"solr_stream\": _solr_base + \"stream\",\n  \"analytics_src\": \"https://metrics.isample.xyz/js/plausible.js\",\n  \"analytics_domain\": \"isamples.org\"\n
+}" | tee ./src/config.js
```
Comment (qgan7125) [inline isb/Dockerfile:19]: Change from `json` to `js`. Using base URL to set up different APIs.

File: isb/Dockerfile [hunk 2]
```diff
@@ -15,7 +15,7 @@ COPY ./isamples_webui/ ./
 ARG ISB_SITEMAP_PREFIX=https://mars.cyverse.org
 ARG ANALYTICS_DOMAIN=isamples.org
 # write out the JSON config for the specific solr URL to hit in the webUI
-RUN echo "{\n    \"solr_url\": \"$ISB_SITEMAP_PREFIX/thing/select\",    \"analytics_src\": \"https://metrics.isample.xyz/js/plausible.js\",\n    \"analytics_domain\": \"$ANALYTICS_DOMAIN\"\n}" | tee ./src/config.json
+RUN echo "const _solr_base = \"$ISB_SITEMAP_PREFIX/thing/\";\n\nconst config = {\n  \"solr_url\": _solr_base + \"select\",\n  \"solr_stream\": _solr_base + \"stream\",\n  \"analytics_src\": \"https://metrics.isample.xyz/js/plausible.js\",\n  \"analytics_domain\": \"isamples.org\"\n};\n\nexport default config;" | tee ./src/config.js
```
Comment (qgan7125) [inline isb/Dockerfile:18]: The new string of `config.js`. Using export default config in the last to solve the bug in `npm run build --if-present`. @dannymandel I have a question. Should I add `isamples_central` between `ISB_SITEMAP_PREFIX` and `/thing`?

Comment (dannymandel) [reply isb/Dockerfile:18]: No, `ISB_SITEMAP_PREFIX` would contain `isamples_central` as part of the string if we were building for an `isamples_central` configuration.

**Comment by dannymandel** [comment]
This looks good to me.  Were you able to test it in conjunction with your other PR?

**Review by dannymandel — APPROVED** [review]

**Comment by qgan7125** [comment]
Not yet, I still cannot run docker build locally.

**Pull request merged**